Products/CMFPlone/browser/contact_info.py

```diff
@@ -67,12 +67,13 @@ def send_message(self, data):
 
         try:
             # This actually sends out the mail
-            host.send(
+            host.secureSend(
```

Comment:
because send doesn't allow setting headers. It's unfortunate the API doesn't allow setting headers of the message.